### ISSUE TITLE
Create state for Client to equivocate "read by source"

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-ux#100]
## Problem
Feature exists in Web UI but needs to exist in Client UI.

## Solution
[Client Repo Issue](https://github.com/freedomofpress/securedrop-client/issues/889)'s blurb summarizes it well:

> There is a somewhat obscure feature in the classic SecureDrop experience, where a journalist reply will transition to "read" in the Journalist Interface once it has been deleted by the source:
> 
> Screenshot_2020-03-06_15-29-29
> Screenshot_2020-03-06_15-29-44
> 
> The client should have feature parity with the Journalist Interface, so we should aim to preserve this behavior in the client, or remove the feature altogether. This is distinct from "read/unread" or "seen/unseen" status, which indicates whether a journalist has read a message or seen a document (#187).

**Comment by ninavizz** [comment]
@martinshelton @eloquence This feature w/in the JWS had not been on my radar to inquire about with the Pilot research, but is something I'd like to ask journalist users about at some point... and, incorporate into design stuff in the near future.

**Comment by ninavizz** [comment]
Tagging child issue w/in #102

**Comment by eloquence** [comment]
Yes, I am definitely interested in whether journalists are even aware that this functionality exists; it's super subtle right now, so I doubt it. However, we can make it less subtle in the client, and it does it seem like a potentially useful feature -- provided sources do follow our instructions and delete replies (which serves as the "read receipt" for the journalist).

**Comment by ninavizz** [comment]
Derp, we need to start asking journalist users about this.
